<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vister</class>
 <widget class="QDialog" name="Vister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>疫情地图管理系统</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>131</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GUI/UI_login_admin_login_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UI_login_guest_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/GUI/UI_login_guest_bg.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GUI/UI_login_btn_quit_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>565</y>
     <width>131</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GUI/UI_login_admin_back_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>UI_login_guest_bg</zorder>
  <zorder>QuitButton</zorder>
  <zorder>exitButton</zorder>
  <zorder>loginButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
